packages/features/handleMarkNoShow.ts

```diff
@@ -113,6 +117,83 @@ const handleMarkNoShow = async ({
         ...(platformClientParams ? platformClientParams : {}),
       });
 
+      const booking = await prisma.booking.findUnique({
+        where: { uid: bookingUid },
+        include: {
+          eventType: {
+            include: {
+              owner: true,
+              team: {
+                select: {
+                  parentId: true,
+                },
+              },
+            },
+          },
+          attendees: true,
+          user: true,
```

Comment:
We should explicitly select the fields we need from user and attendees instead of including everything

Reply:
same here: https://github.com/calcom/cal.com/pull/23068/files#r2288376065